frontend/packages/core/src/NPS/emojiRatings.tsx

```diff
@@ -27,7 +32,13 @@ const getKey = (map, val): string => Object.keys(map).find(key => map[key] === v
  * @param setRating function which will return the given selected rating
  * @returns rendered EmojiRatings component
  */
-const EmojiRatings = ({ ratings = [], setRating }: EmojiRatingsProps) => {
+const EmojiRatings = ({
+  ratings = [],
+  setRating,
+  placement = "top",
+  size = "small",
+  emojiSize = "medium",
```

Comment:
nit, since we're using the medium emoji size now, can we make the button size smaller as well? i think it's getting the large hover space from the button size

<img width="225" alt="Screen Shot 2022-01-24 at 9 52 58 AM" src="https://user-images.githubusercontent.com/39421794/150805954-2002a969-c041-4fa4-abfc-6fa380fc4970.png">

Reply:
Yeah, I had to override the padding for that size since the default 12px padding is just too much for that size